### PR TITLE
Replace all references to the git branch 'master' with 'main'

### PR DESCRIPTION
**Pull request by mo-nathan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,8 +53,8 @@ See [LICENSE][12] for further details.
...
 [1]: <https://secure.travis-ci.org/MushroomObserver/mushroom-observer.png>
 [2]: <http://travis-ci.org/MushroomObserver/mushroom-observer>
-[3]: <https://coveralls.io/repos/MushroomObserver/mushroom-observer/badge.png?branch=master>
-[4]: <https://coveralls.io/r/MushroomObserver/mushroom-observer?branch=master>
+[3]: <https://coveralls.io/repos/MushroomObserver/mushroom-observer/badge.png?branch=main>
+[4]: <https://coveralls.io/r/MushroomObserver/mushroom-observer?branch=main>
 [5]: <https://codeclimate.com/github/MushroomObserver/mushroom-observer.png>
 [6]: <https://codeclimate.com/github/MushroomObserver/mushroom-observer>
-[12]: <https://github.com/MushroomObserver/mushroom-observer/blob/master/LICENSE>
+[12]: <https://github.com/MushroomObserver/mushroom-observer/blob/main/LICENSE>
```
Comment (JoeCohen) [inline README.md:60]: Would relative links be better?  See https://github.blog/2013-01-31-relative-links-in-markup-files/
```suggestion
[12]: <LICENSE>
```

File: README_API.md [hunk 2]
```diff
@@ -209,8 +209,8 @@ ORDER BY observations.id ASC
 ```
 See also the database diagram here:
 
-* <https://github.com/MushroomObserver/mushroom-observer/blob/master/DATA_STRUCTURE.gif>
+* <https://github.com/MushroomObserver/mushroom-observer/blob/main/DATA_STRUCTURE.gif>
```
Comment (JoeCohen) [inline README_API.md:212]:
```suggestion
* [DATA_STRUCTURE.gif](DATA_STRUCTURE.gif)
```

File: README_API.md [hunk 3]
```diff
@@ -209,8 +209,8 @@ ORDER BY observations.id ASC
...
 
 and the database schema here:
 
-* <https://github.com/MushroomObserver/mushroom-observer/blob/master/db/schema.rb>
+* <https://github.com/MushroomObserver/mushroom-observer/blob/main/db/schema.rb>
```
Comment (JoeCohen) [inline README_API.md:216]:
```suggestion
* [db/schema.rb](db/schema.rb)
```

**Review by JoeCohen — DISMISSED** [review]
Thanks for doing this. It looks good and it will nice to get this out of the way.
My only suggestion was using markdown relative links.

**Comment by pellaea** [comment]
Do we need to do that now?  Are now officially on main?

**Comment by mo-nathan** [comment]
Had to merge the PR and change the default branch, but now both of those
are done so we should be good to go.

**Pull request merged**